docusaurus/docs/React/components/contexts/channel-state-context.mdx

```diff
@@ -57,6 +57,15 @@ If true, chat users will be able to drag and drop file uploads to the entire cha
 | ------- | ------- |
 | boolean | false   |
 
+### enrichURLForPreview
+
+A global flag to toggle the URL enrichment and link previews in `MessageInput`. By default, the feature is disabled. Can be overridden on Thread and MessageList level through `additionalMessageInputProps`
```

Comment:
```suggestion
A global flag to toggle the URL enrichment and link previews in `MessageInput`. By default, the feature is disabled. It can be overridden on Thread and MessageList level through `additionalMessageInputProps`
```